Source/ISHRemote/Trisoft.ISHRemote/Objects/Public/IshTypeFieldDefinition.cs

```diff
@@ -55,31 +61,34 @@ public class IshTypeFieldDefinition : IComparable
         public bool AllowOnCreate { get; internal set; }
         public bool AllowOnUpdate { get; internal set; }
         public bool AllowOnSearch { get; internal set; }
+        public bool AllowOnSmartTagging { get; internal set; }
         public bool IsSystem { get; internal set; }
         public bool IsBasic { get; internal set; }
         public bool IsDescriptive { get; internal set; }
         public string Label { get; internal set; }
         public string Description { get; internal set; }
         public List<Enumerations.ISHType> ReferenceType { get; internal set; }
         public string ReferenceLov { get; internal set; }
-
+        public string ReferenceMetadataBinding { get; internal set; }
 
         /// <summary>
-        /// PS1XML Shorthand notation of DataType-ReferenceType-ReferenceLov properties
+        /// PS1XML Shorthand notation of DataType-(ReferenceMetadataBinding/ReferenceLov/ReferenceType) properties
         /// </summary>
-        public string Type
+        public string TypeSource
         {
             get
             {
                 switch (DataType)
                 {
+                    case Enumerations.DataType.ISHMetadataBinding:
+                        return ReferenceMetadataBinding;
                     case Enumerations.DataType.ISHLov:
                         return ReferenceLov;
                     case Enumerations.DataType.ISHType:
                         // return sorted to avoid string compare issues when database seq differed
                         return string.Join(",",ReferenceType.OrderBy(q => q).ToList());
                     default:
-                        return DataType.ToString();
+                        return String.Empty;
```

Comment:
This changed because?

Reply:
Because that is how I read image attached where "date/number/string we see empty". We can discuss this again if there is a value to see "String" for a "String"

Reply:
In the past, it was never rendered in `Out-GridView` and the likes... I might be a bit confused here
* DataType column, which is filled with Type object property remains as is, besides adding ISHMetadataBinding
* New column DataSource, which is filled by TypeSource can indeed return empty, because it would be superfluous to have String and String next to each other...

Reply:
Correct, DataType column filled with Type and only gets new ISHMetadataBinding
DataSource is filled with TypeSource(former Type) now returns ReferenceMetadataBinding for ISHMetadataBinding and empty string for all unsupported values (so not ISHLov or ISHMetadataBinding or ISHType). So a second 'breaking' change described in draft of https://github.com/sdl/ISHRemote/releases/